pkg/console/subresource/consoleserver/types.go

```diff
@@ -18,8 +18,8 @@ type Config struct {
 	Auth           `yaml:"auth"`
 	Customization  `yaml:"customization"`
 	Providers      `yaml:"providers"`
-	MonitoringInfo `yaml:"monitoringInfo,omitempty"`
-	LoggingInfo    `yaml:"loggingInfo,omitempty"`
+	MonitoringInfo `yaml:"monitoringInfo"`
```

Comment:
I would leave of `omitempty` here, as the others props at this level don't have it.

Reply:
These all should have it, but it was never fixed.

Reply:
> These all should have it, but it was never fixed.

We'd need to change them to pointers for `omitempty` to work iirc